docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### routine_load_thread_pool_size
+
+- **Default**: 10
+- **Unit**: N/A
+- **Description**: The thread pool size for Routine Load on each BE. Since v3.1.0, this parameter is deprecated. The thread pool size for Routine Load on each BE is now controlled by the FE dynamic parameter max_routine_load_task_num_per_be.
+
+#### brpc_max_body_size
+
+- **Default**: 2147483648
+- **Unit**: Byte
+- **Description**: The maximum body size of a BRPC.
+
+#### tablet_map_shard_size
+
+- **Default**: 32
+- **Unit**: N/A
+- **Description**: The tablet map shard size. The value must be a power of two.
+
+#### enable_bitmap_union_disk_format_with_set
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the new storage format of the BITMAP type, which can improve the performance of bitmap_union. TRUE indicates enabling the new storage format, and FALSE indicates disabling it.
+
+#### mem_limit
+
+- **Default**: 90%
+- **Unit**: N/A
+- **Description**: BE process memory upper limit. You can set it as a percentage ("80%") or a physical limit ("100GB").
+
+#### flush_thread_num_per_store
+
+- **Default**: 2
+- **Unit**: N/A
+- **Description**: Number of threads that are used for flushing MemTable in each store.
+
+#### block_cache_enable
+
+- **Default**: false
+- **Unit**: N/A
+- **Description**: Whether to enable Data Cache. TRUE indicates Data Cache is enabled, and FALSE indicates Data Cache is disabled.
+
+#### block_cache_disk_path
+
+- **Default**: N/A
+- **Unit**: N/A
+- **Description**: The paths of disks. We recommend that the number of paths you configure for this parameter is the same as the number of disks on your BE machine. Multiple paths need to be separated with semicolons (;). After you add this parameter, StarRocks automatically creates a file named cachelib_data to cache blocks.
+
+#### block_cache_meta_path
+
+- **Default**: N/A
+- **Unit**: N/A
+- **Description**: The storage path of block metadata. You can customize the storage path. We recommend that you store the metadata under the $STARROCKS_HOME path.
+
+#### block_cache_mem_size
+
+- **Default**: 2147483648
+- **Unit**: Bytes
+- **Description**: The maximum amount of data that can be cached in memory. Unit: bytes. The default value is 2147483648, which is 2 GB. We recommend that you set the value of this parameter to at least 20 GB. If StarRocks reads a large amount of data from disks after Data Cache is enabled, consider increasing the value.
+
+#### block_cache_disk_size
+
+- **Default**: 0
+- **Unit**: Bytes
+- **Description**: The maximum amount of data that can be cached on a single disk. For example, if you configure two disk paths for the block_cache_disk_path parameter and set the value of the block_cache_disk_size parameter as 21474836480 (20 GB), a maximum of 40 GB data can be cached on these two disks. The default value is 0, which indicates that only memory is used to cache data. Unit: bytes.
```

Comment:
See to all parameter values starting from here till the end of this doc and make sure they are wrapped in code format.